test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -684,157 +685,25 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
-
-			ginkgo.By("Creating Internet gateway")
-			igw, _ := shared.CreateInternetGateway(e2eCtx, clusterName+"-igw")
-			Expect(igw).NotTo(BeNil())
-			igwID := *igw.InternetGatewayId
-			igwA, _ := shared.AttachInternetGateway(e2eCtx, igwID, vpcID)
-			Expect(igwA).To(BeTrue())
-
-			ginkgo.By("Allocating Elastic IP")
-			eip, _ := shared.AllocateAddress(e2eCtx, clusterName+"-eip")
-			Expect(eip).NotTo(BeNil())
-			allocationID := *eip.AllocationId
-
-			ginkgo.By("Creating NAT gateway")
-			ngw, _ := shared.CreateNatGateway(e2eCtx, clusterName+"-nat", "public", allocationID, pubSubnetID)
-			Expect(ngw).NotTo(BeNil())
-			ngwID := *ngw.NatGatewayId
-			shared.WaitForNatGatewayState(e2eCtx, ngwID, 180, "available")
-
-			ginkgo.By("Creating route tables")
-			pubRoute, _ := shared.CreateRouteTable(e2eCtx, clusterName+"-rt-public", vpcID)
-			priRoute, _ := shared.CreateRouteTable(e2eCtx, clusterName+"-rt-private", vpcID)
-
-			ginkgo.By("Creating associating routes")
-			pubRouteID := *pubRoute.RouteTableId
-			priRouteID := *priRoute.RouteTableId
-			pubAssocRT, _ := shared.AssociateRouteTable(e2eCtx, pubRouteID, pubSubnetID)
-			priAssocRT, _ := shared.AssociateRouteTable(e2eCtx, priRouteID, priSubnetID)
-
-			ginkgo.By("Creating routes")
-			shared.CreateRoute(e2eCtx, pubRouteID, "0.0.0.0/0", nil, igw.InternetGatewayId, nil)
-			shared.CreateRoute(e2eCtx, priRouteID, "0.0.0.0/0", ngw.NatGatewayId, nil, nil)
-
-			configCluster := defaultConfigCluster(clusterName, namespace.Name)
-			configCluster.WorkerMachineCount = pointer.Int64Ptr(1)
-			configCluster.Flavor = "external-securitygroups"
-			cluster, md, _ := createCluster(ctx, configCluster, result)
-
-			workerMachines := framework.GetMachinesByMachineDeployments(ctx, framework.GetMachinesByMachineDeploymentsInput{
-				Lister:            e2eCtx.Environment.BootstrapClusterProxy.GetClient(),
-				ClusterName:       clusterName,
-				Namespace:         namespace.Name,
-				MachineDeployment: *md[0],
-			})
-			controlPlaneMachines := framework.GetControlPlaneMachinesByCluster(ctx, framework.GetControlPlaneMachinesByClusterInput{
-				Lister:      e2eCtx.Environment.BootstrapClusterProxy.GetClient(),
-				ClusterName: clusterName,
-				Namespace:   namespace.Name,
-			})
-			Expect(len(workerMachines)).To(Equal(1))
-			Expect(len(controlPlaneMachines)).To(Equal(1))
-
-			deleteCluster(ctx, cluster)
-
-			ginkgo.By("Disassociating route tables")
-			pubAssocID := *pubAssocRT.AssociationId
-			priAssocID := *priAssocRT.AssociationId
-			Expect(shared.DisassociateRouteTable(e2eCtx, pubAssocID)).To(BeTrue())
-			Expect(shared.DisassociateRouteTable(e2eCtx, priAssocID)).To(BeTrue())
-
-			ginkgo.By("Deleting route tables")
-			Expect(shared.DeleteRouteTable(e2eCtx, priRouteID)).To(BeTrue())
-			Expect(shared.DeleteRouteTable(e2eCtx, pubRouteID)).To(BeTrue())
+	ginkgo.Describe("External infrastructure, VPC peering, internal ELB and private subnet use only", func() {
```

Comment:
Would be worth to mention BYO SG is also included in this test either in the name or comment.